docs/tutorials/kuai-cli-deploy-contract.md

```diff
@@ -0,0 +1,158 @@
+# How to deploy contracts with kuai-cli
+
+## deploy contract workflow by builtin signer (directly deploy):
+
+1. Create a new conrtact, skip this step if the contract already exists ([How to import exists contract](#how-to-import-contract-to-kuai-project))
+
+- `kuai contract new --name [contract-name]`
+
+```bash
+$ kuai contract new --name always-success
+
+New contract "always-success"
+    Created binary (application) `always-success` package
+Rewrite Cargo.toml
+Rewrite capsule.toml
+Done
+```
```

Comment:
```suggestion
  ```bash
  $ kuai contract new --name always-success
  
  # New contract "always-success"
  #     Created binary (application) `always-success` package
  # Rewrite Cargo.toml
  # Rewrite capsule.toml
  # Done
  ```
```

Reply:
This update makes indention correct, and `#` makes `outputs` commented out so users can copy-paste it directly without deleting the `outputs` in the terminal

Reply:
@PainterPuppets please check these suggestions